apps/src/templates/sectionAssessments/FreeResponsesSurveyTable.story.jsx

```diff
@@ -2,19 +2,19 @@ import React from 'react';
 import FreeResponsesSurveyTable from './FreeResponsesSurveyTable';
 import {surveyOne, surveyTwo} from './assessmentsTestHelpers';
 
-export default storybook => {
-  return storybook
-    .storiesOf('SectionAssessments/FreeResponsesSurveyTable', module)
-    .addStoryTable([
-      {
-        name: 'Free responses for survey 1',
-        description: 'Display anonymous responses',
-        story: () => <FreeResponsesSurveyTable freeResponses={surveyOne} />
-      },
-      {
-        name: 'Free responses for survey 2',
```

Comment:
This feels like something that should be unit tested instead of storybook tested (ie we want to check that the table isn't displayed). So if there is a test that covers this already, I'm down to delete this

Reply:
I totally agree.. I can't find a unit test for this or for the container that uses it😬 So does that mean we keep this for now and it's just a followup tech debt to write unit tests in this area and remove the stories?

Reply:
I would leave the storybook then file a ticket somewhere I guess to write unit tests. (I'm actually going to move this to Slack because I'm curious what everyone would like to do about this scenario)

Reply:
Until we come to a resolution as a team: I created a ticket and put it in our "Candidates for Up Next" category. https://codedotorg.atlassian.net/browse/TEACH-105